### ISSUE TITLE
Clicking a figure, going back to the catalog tab, and then clicking the same figure doesn't do anything

### ISSUE THREAD

**Issue by daattali** [jennybc/r-graph-catalog#15]
If you try to click on a figure that's already the last viewed figure, nothing happens (you don't get automatically moved to the figure tab)

**Comment by jennybc** [comment]
😥

**Comment by jennybc** [comment]
Do you know how to fix that?

**Comment by daattali** [comment]
I'm pretty sure [this is the cause](https://github.com/rstudio/shiny/issues/928)

There's no super duper clean fix for it, but it can done.... semi-hackily...

It's not very high pri, is it?  I don't want to deal with that right now, remind me in a month if you still care about it

**Comment by jennybc** [comment]
Well I guess it can wait. You can always go from fig x to catalog to fig x by visiting fig y somewhere along the way.

Is this a sign we should do this the other way you mentioned? Via the URL?

**Comment by daattali** [comment]
Not really. The only way you could use the URL to fix this is if clicking on each image actually went to a new URL.  But that would mean that every click on an image would cause the entire app to re-load, which isn't nice.  The way the URL thing works right now is that we just add some text to the URL, but there is no page reload.  I'm not sure if that makes sense?  It's just a common problem shiny apps that use javascript actions face, it's not indicative of a poor app design
